MagnetScannerIOS/NetworkResolver.swift

```diff
@@ -15,11 +15,11 @@ import SwiftyJSON
 //
 public class NetworkResolver {
   static let API_END_POINT: String = "https://tengam.org/content/v1/search/beacons/"
-  static let RADIUS: Int = 50
+  static let RADIUS: Int = 100
 
   static func resolveLocation(lat: Double, lon: Double, callback: (Array<JSON> -> Void)!) {
```

Comment:
NIT Naming: We're not actually resolving the location, we're resolving the item from the location. Something like `fetchItem` or `resolveItem` with `lat` and `lon` are arguments is clearer to me.